examples/README.rst

```diff
@@ -85,83 +87,60 @@ Step-by-Step Weather Data Example
 .. code:: scala
 
     import org.apache.spark.h2o._
-    val h2oContext = H2OContext.getOrCreate(spark)
-    import h2oContext._
-    import h2oContext.implicits._
+    val hc = H2OContext.getOrCreate(spark)
+    import hc.implicits._
+    import spark.implicits._
 
-4.  Load weather data for Chicago international airport (ORD), with help
-    from the RDD API:
+4.  Load weather data for Chicago international airport (ORD):
 
 .. code:: scala
 
-    import org.apache.spark.examples.h2o._
     val weatherDataFile = "examples/smalldata/chicago/Chicago_Ohare_International_Airport.csv"
-    val wrawdata = spark.sparkContext.textFile(weatherDataFile,3).cache()
-    val weatherTable = wrawdata.map(_.split(",")).map(row => WeatherParse(row)).filter(!_.isWrongRow())
+    val weatherTable = spark.read.option("header", "true").option("inferSchema", "true").csv(weatherDataFile).withColumn("Date", to_date('Date, "MM/dd/yyyy")).withColumn("Year", year('Date)).withColumn("Month", month('Date)).withColumn("DayofMonth", dayofmonth('Date))
```

Comment:
WDYT about referomating the logic into multiple lines. (via the `select` statement or user can use `:paste` mode.

Reply:
👍